src/test/java/com/github/underscore/CollectionsTest.java

```diff
@@ -1031,6 +1030,26 @@ public void contains() {
         assertFalse(result3);
         final boolean result4 = U.contains(asList(1, 2, null), null);
         assertTrue(result4);
+        final boolean result5 = U.containsAtLeast(asList(1, 2, 2), 2, 2);
+        assertTrue(result5);
+        final boolean result6 = U.containsAtLeast(asList(1, 2, 2), 2, 3);
+        assertFalse(result6);
+        final boolean result7 = U.containsAtMost(asList(1, 2, 2), 2, 3);
+        assertTrue(result7);
+        final boolean result8 = U.containsAtMost(asList(1, 2, 2), 2, 1);
+        assertFalse(result8);
+        final boolean result9 = new U<Integer>(asList(1, 2, 2)).containsAtLeast(2, 2);
+        assertTrue(result9);
+        final boolean result10 =  new U<Integer>(asList(1, 2, 2)).containsAtLeast(2, 3);
+        assertFalse(result10);
+        final boolean result11 = new U<Integer>(asList(1, 2, 2)).containsAtMost(3, 2);
+        assertTrue(result11);
+        final boolean result12 =  new U<Integer>(asList(1, 2, 2)).containsAtMost(2, 1);
+        assertFalse(result12);
+        final boolean result13 =  U.containsAtMost(asList(null, null, 2), null, 2);
+        assertTrue(result13);
+        final boolean result14 =  U.containsAtLeast(asList(null, null, 2), null, 2);
```

Comment:
Good. We have a test case `null` in list and `null` value. It may be one more test case `null` in list and not null value.